node/tests/p2p_test.go

```diff
@@ -90,7 +90,13 @@ func TestBootstrapNodesFromBridgeNode(t *testing.T) {
 	sw := swamp.NewSwamp(t)
 	cfg := node.DefaultConfig(node.Bridge)
 	cfg.P2P.Bootstrapper = true
-	bridge := sw.NewBridgeNode(node.WithConfig(cfg), node.WithRefreshRoutingTablePeriod(time.Second*30))
+	const defaultTimeInterval = time.Second * 10
+
+	bridge := sw.NewBridgeNode(node.WithConfig(cfg),
+		node.WithRefreshRoutingTablePeriod(defaultTimeInterval),
+		node.WithDiscoveryInterval(defaultTimeInterval),
+		node.WithAdvertiseInterval(defaultTimeInterval),
```

Comment:
nit but you can extract this into a var discOpts = []opt{} and use it throughout the file.